x-pack/plugins/maps/common/elasticsearch_util/elasticsearch_geo_utils.ts

```diff
@@ -381,14 +381,7 @@ export function createSpatialFilterWithGeometry({
   relation: ES_SPATIAL_RELATIONS;
 }): GeoFilter {
   ensureGeoField(geoFieldType);
-
-  const isGeoPoint = geoFieldType === ES_GEO_FIELD_TYPE.GEO_POINT;
-
-  const relationLabel = isGeoPoint
-    ? i18n.translate('xpack.maps.es_geo_utils.shapeFilter.geoPointRelationLabel', {
-        defaultMessage: 'in',
-      })
-    : getEsSpatialRelationLabel(relation);
+  const relationLabel = getEsSpatialRelationLabel(relation);
```

Comment:
Nit: Since relationLabel is only used in a single location, can you just use `getEsSpatialRelationLabel(relation)` inline instead of storing in local variable.